examples/jsm/nodes/math/MathNode.js

```diff
@@ -301,6 +303,12 @@ export const pow3 = nodeProxy( MathNode, MathNode.POW, 3 );
 export const pow4 = nodeProxy( MathNode, MathNode.POW, 4 );
 export const transformDirection = nodeProxy( MathNode, MathNode.TRANSFORM_DIRECTION );
 
+// remapping functions https://iquilezles.org/articles/functions/
+export const parabola = ( x, k ) => pow( mul( 4.0, x.mul( sub( 1.0, x ) ) ), k );
```

Comment:
Hm.. Maybe we can put these functions into a different node file? `MathNode` feels like the place for the most basic maths (functions supported OOTB in GLSL/WGSL mostly), not more complex functions like these...